lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -1765,7 +1765,7 @@ void ByteCodeGenerator::InitScopeSlotArray(FuncInfo * funcInfo)
 {
     // Record slots info for ScopeSlots/ScopeObject.
     uint scopeSlotCount = funcInfo->bodyScope->GetScopeSlotCount();
-    uint scopeSlotCountForParamScope = funcInfo->paramScope ? funcInfo->paramScope->GetScopeSlotCount() : 0;
+    uint scopeSlotCountForParamScope = funcInfo->paramScope != nullptr && !funcInfo->paramScope->GetCanMergeWithBodyScope() ? funcInfo->paramScope->GetScopeSlotCount() : 0;
```

Comment:
why paramScope->GetScopeSlotCount not be 0 when GetCanMergeWithbodyScope is true ? I believe the paramScope->GetScopeSlotCount will be populated only when the scope is split.